apps/heft/src/pluginFramework/HeftTask.ts

```diff
@@ -72,7 +72,15 @@ function _getNodeServicePluginDefinition(): HeftTaskPluginDefinition {
     _nodeServicePluginDefinition = HeftTaskPluginDefinition.loadFromObject({
       heftPluginDefinitionJson: {
         pluginName: 'node-service-plugin',
-        entryPoint: './lib/plugins/NodeServicePlugin'
+        entryPoint: './lib/plugins/NodeServicePlugin',
+        parameters: [
+          {
+            longName: '--serve',
+            parameterKind: 'flag',
+            description:
+              'Start a local web server for testing purposes.  This parameter is only available when running in watch mode.'
```

Comment:
What does it mean to invoke a node service via the NodeServicePlugin when not running in watch or serve mode?

Reply:
If you did not specify `--serve` AND you are not running in `build-watch` mode, then you are simply launching the service, as might be done in production.  By Rush Stack convention, you would do that like this:

```
$ rushx serve
```
...which by convention invokes this:

**package.json**
```js
. . .
"scripts": {
  "serve": "node lib/start.js",
  . . .
}
```

Consequently the [node-service.json](https://rushstack.io/pages/heft_configs/node-service_json/) configuration defaults to invoking your `serve` script:

```js
  /**
   * Specifies the name of a "scripts" command from the project's package.json file.
   * When "heft start" is invoked, it will use this shell command to launch the
   * service process.
   *
   * Default value: "serve"
   */
  // "commandName": "serve",
```